app/src/Entity/LdapConnector.php

```diff
@@ -52,6 +52,12 @@ class LdapConnector extends Connector
     #[ORM\Column(length: 255, nullable: true)]
     private ?string $ldapSharedWithField = null;
 
+    #[ORM\Column(type: 'string', length: 255, nullable: true)]
+    private ?string $ldapEncryption = 'none';
+
+    #[ORM\Column(type: 'integer', nullable: true)]
+    private ?int $ldapVersion = 3;
```

Comment:
Ces attributs ne doivent surtout pas être nullable, à la place on va définir des valeurs par défaut avec les valeurs qui étaient utilisées auparavant (= celles par défaut quand on appelle `Ldap::create()`, c’est-à-dire [`none` et `3`](https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/Ldap/Adapter/AbstractConnection.php#L35-L41)) :

```suggestion
    #[ORM\Column(type: 'string', length: 255, options: ['default' => 'none'])]
    private string $ldapEncryption = 'none';

    #[ORM\Column(type: 'integer', options: ['default' => 3])]
    private int $ldapVersion = 3;
```

Par contre ça signifie qu’il faut regénérer la migration.